modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/search_container_select.js

```diff
@@ -206,9 +206,9 @@ AUI.add(
 				},
 
 				_isActionUrl(url) {
-					var uri = new A.Url(url);
+					const uri = new URL(url);
 
-					return Number(uri.getParameter('p_p_lifecycle')) === 1;
+					return Number(uri.searchParams.get('p_p_lifecycle')) === 1;
```

Comment:
Just a consideration here, if this code would be backported We must provide a polyfill for searchParams methods in older versions or use another solution. See [searchParams](https://caniuse.com/mdn-api_url_searchparams) browser support.

Reply:
Remember we already have the [compatibility-ie](https://github.com/liferay/liferay-portal/tree/7.3.x/modules/apps/frontend-compatibility/frontend-compatibility-ie) module in `7.3.x` which [includes among others](https://github.com/liferay/liferay-portal/blob/7.3.x/modules/apps/frontend-compatibility/frontend-compatibility-ie/src/main/java/com/liferay/frontend/compatibility/ie/internal/servlet/taglib/IETopHeadDynamicInclude.java#L97-L99), the [core-js](https://github.com/zloirock/core-js/blob/master/README.md) bundle